cla_signers.py

```diff
@@ -72,12 +73,14 @@ def ValidateAccounts(accounts):
         if not account:
             return (False, ['Invalid empty account found'])
         if sorted(account.keys()) != sorted(accounts_keys):
-            return (False, ['The only allowed and required keys for people/bot accounts are: %s.' % str(accounts_keys),
+            return (False, ['The only allowed and required keys for people/bot'
+                            ' accounts are: %s.' % str(accounts_keys),
```

Comment:
In other cases of line continuation, the space is on the previous line. Care to move it to the previous line for consistency?

Reply:
Sure. that space put the previous line over the limit, but I'll just make it three lines. Let me know if there's anything else, otherwise I'm fine with you merging this.

Reply:
Oh, sorry, didn't realize that! We could convert `s/and/&/` to shrink it to fit, but no need to worry about it; I'll merge it as-is. Thanks!